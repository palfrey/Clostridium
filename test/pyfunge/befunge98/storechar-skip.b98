6a*1-sv"ti serongi dna nolocimes stup s :DOOG",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,@;v
,,,,,, ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"BAD: s puts semicolon but does not ignore it"<@,,,,,,
      <@,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"BAD: s doesn't put semicolon at all"
