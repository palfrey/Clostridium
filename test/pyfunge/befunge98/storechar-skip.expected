GOOD: s puts semicolon and ignores it
